pygmt/tests/test_figure.py

```diff
@@ -1,6 +1,6 @@
 """
-Test the behaviors of the Figure class
-Doesn't include the plotting commands, which have their own test files.
+Test the behaviors of the Figure class Doesn't include the plotting commands,
+which have their own test files.
```

Comment:
```suggestion
Tests the behavior of the Figure class.

Doesn't include the plotting commands which have their own test files.
```
One last typo to fix :slightly_smiling_face: